FullStoryMiddleware/FullStoryMiddleware.h

```diff
@@ -12,6 +12,13 @@
 #import <Foundation/Foundation.h>
 #import <Analytics/SEGMiddleware.h>
 
+//! Project version number for FullStoryMiddleware.
+FOUNDATION_EXPORT double FullStoryMiddlewareVersionNumber;
+
+//! Project version string for FullStoryMiddleware.
+FOUNDATION_EXPORT const unsigned char FullStoryMiddlewareVersionString[];
+
+
 @interface FullStoryMiddleware : NSObject<SEGMiddleware>
 
 @property (strong, nonatomic) NSArray* allowlistEvents;
```

Comment:
While we're restructuring the project...
```suggestion
@property (strong, nonatomic) NSArray<NSString *>* allowlistEvents;
```